plot/plot.app.js

```diff
@@ -412,8 +411,18 @@
                         defer.resolve(plot_values);// TODO: figure out how to return an error to catch clause without breaking code
                         return defer.promise;
                     }
-
-                    server.http.get(uri).then(function(response) {
+                    var headers = {};
+                    headers[ERMrest.contextHeaderName]=ConfigUtils.getContextHeaderParams();
+                    if(uriParams[uriParams.indexOf("attributegroup")+1])
+                        headers[ERMrest.contextHeaderName].schema_table=uriParams[uriParams.indexOf("attributegroup")+1]
```

Comment:
`schema_table` is just the `schema:table` defined in the request. If an alias is defined, that alias needs to be removed. `M:=schema:table` should be simplified to `schema:table`.

Reply:
Made required changes where ever applicable